api/services/ocean/gke/schemas/oceanClusterLaunchSpec.yaml

```diff
@@ -27,7 +27,7 @@ properties:
         description: Set root volume size (in GB)
         example: 8
       metadata:
-        type: array of object
+        type: object
```

Comment:
why object and not array?

Reply:
You are correct, this should be array. Will correct